vcsclient/vcsclient.go

```diff
@@ -46,7 +46,7 @@ type VcsClient interface {
 	// repository    - VCS repository name
 	// branch        - VCS branch name
 	// payloadUrl    - URL to send the payload when a webhook event occurs
-	// webhookEvents - PrCreated, PrEdited, or Push
+	// webhookEvents - PrOpened, PrEdited, or Push
```

Comment:
To avoid syncing this comment with the new events every time we add new ones:
```suggestion
	// webhookEvents - The event type
```